docs/DeveloperGuide.md

```diff
@@ -559,14 +559,39 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
-
 * *a. User input the wrong command.
 
 	* *a1. HelloFile shows the error message and direct the user to enter the user help command.
 
 	    Use case ends.
 <br />
 
+**UseCase ID: UC08**
+
+**Use Case: Find a tag/label**
+
+**MSS**
+
+1. User request to find a tag/label from the given keyword
+2. HelloFile recognize the keyword is associated with existing tag/label.
+3. HelloFile show the result to the user 
+    
+    Use case ends
+    
+**Extensions**
+
+* *2a. HelloFile cannot associate keyword with any tag/label.
+
+    * *2a1. HelloFile will show nothing on the list.
```

Comment:
I think "HelloFile shows nothing" is probably more suitable here?

Reply:
Okay will fix